rplugin/python3/LanguageClient/LanguageClient.py

```diff
@@ -49,8 +49,9 @@ def __init__(self, nvim):
         self.hlsid = None
         self.signid = 0
         type(self)._instance = self
-        self.serverCommands = self.nvim.eval(
+        self.serverCommands = nvim.eval(
                 "get(g:, 'LanguageClient_serverCommands', {})")
+        self.skip_threshold = 0
```

Comment:
I'd prefer variables sync with vim side instead of inventing another name. What about change it to `changeThreshold`?